src/System/TimedScopes/ITimedScopeLogger.cs

```diff
@@ -1,6 +1,8 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license.
 
+using Microsoft.Omex.System.Logging;
```

Comment:
>Microsoft.Omex.System.Logging [](start = 6, length = 29)

Is this required? I don't see any new dependencies in the class, so I'm not sure why this would be necessary (unless something moved to a different namespace)